rpc/jsonrpc/net_api.go

```diff
@@ -27,9 +27,9 @@ import (
 
 // NetAPI the interface for the net_ RPC commands
 type NetAPI interface {
-	Listening(_ context.Context) (bool, error)
+	// Listening(_ context.Context) (bool, error)
```

Comment:
why that?

Reply:
some methods are disabled/unavailable in arbitrum. With this we will have same result as arbitrum

Reply:
lets then add check on arbitrum config and ignore net_ namespace